TypeCobol/Compiler/Nodes/Node.cs

```diff
@@ -403,7 +403,7 @@ public virtual List<string> VisualQualifiedNameWithoutProgram
                 if (_root != null) return _root;
                 var current = this;
                 while (current.Parent != null) current = current.Parent;
-                _root = (SourceFile) current;
+                _root = current as SourceFile;
```

Comment:
In which case, the root Node is not a SourceFile ?

Reply:
I think it was required at some point but not anymore. I reverted this change and no test fail...